docs/src/components/productMaterial/MaterialStyling.tsx

```diff
@@ -235,7 +235,7 @@ export default function MaterialStyling() {
                 overflow: 'auto',
               }}
             >
-              <Box sx={{ position: 'relative' }}>
+              <Box sx={{ position: 'relative', minWidth: 510 }}>
```

Comment:
I wouldn't expect to see a hard-coded width. Isn't there a way to make the highlight follow the width of the code content? Perhaps it's in the wrong container?